boardwalk/README.md

```diff
@@ -9,53 +9,40 @@ Boardwalk is the core's file browser to display the metadata of files stored in
-* Go to the "Credentials" tab. Click on the "Create Credentials" drop down menu and choose "OAuth Client ID".
-* Choose "Web Application" from the menu. Assign it a name. 
-* Under "Authorized JavaScript origins", enter `http://<YOUR_SITE>`. Press Enter. Add a second entry, same as the first one, but use *https* instead of *http*
-* Under "Authorized redirect URIs", enter `http://<YOUR_SITE>/gCallback`. Press Enter. Add a second entry, same as the first one, but use *https* instead of *http*
-* Click "Create". A pop up window will appear with your Google Client ID and Google Client Secret. Save these. If you lose them, you can always go back to the Google Console, and click on your project; the information will be there. Keep these values stored in a safe location. Treat them as you would treat a credit card number. 
-
-**Please note:** at this point, the dashboard only accepts login from emails with a 'ucsc.edu' domain. In the future, it will support different email domains. 
-
-### Development Mode
-
-The boardwalk installer has the option to install boardwalk and its components in either devevelopment or production mode (dev/prod). If you are installing in production, skip this section and head to **Installation Questions**. Otherwise, keep reading.
-
+Deployment is done through the `install_bootstrap` script on the root directory. It will ask you a series of question to collect the information required to install boardwalk and its components.
+
+### Create a Google Oauth2 app
+
+Before proceeding, you must ensure that you have a Google OAuth2 app. Follow the instructions [here](http://bitwiser.in/2015/09/09/add-google-login-in-flask.html#creating-a-google-project) under "Creating A Google Project". 
+
+If you don't want to enable login and token retrieval through the portal, you can just put some random string for the Google Client ID and Google Client Secret. 
+
+Here is a summary of what you need to do in order to create a Google OAuth2 app:
+
+* Go to the [Google Developer Console](https://console.developers.google.com/).
+* On the upper left side of the screen, click on the drop down button.
+* Create a project by clicking on the plus sign in the pop-up window.
+* On the next pop up window, add a name for your project.
+* Once you create it, click on the "Credentials" section on the left hand side.
+* Click on the "OAuth Consent Screen". Fill out a product name and choose an email for the Google Application. Fill the rest of the entries as you see fit for your purposes, or leave them blank, as they are optional. Save it.
+* Go to the "Credentials" tab. Click on the "Create Credentials" drop down menu and choose "OAuth Client ID".
+* Choose "Web Application" from the menu. Assign it a name.
+* Under "Authorized JavaScript origins", enter `http://<YOUR_SITE>`. Press Enter. Add a second entry, same as the first one, but use `https` instead of `http`.
```

Comment:
You should get rid of the http and only give instructions for https. If you try to go to http endpoint, you get redirected to https.